src/test/java/org/breedinginsight/api/auth/rules/ValidUserSecurityRuleIntegrationTest.java

```diff
@@ -39,9 +40,9 @@
 @MicronautTest
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
-public class ValidUserSecurityRuleIntegrationTest {
+public class ValidUserSecurityRuleIntegrationTest extends DatabaseTest {
 
-    private String testUserUUID;
+    String testUserUUID;
```

Comment:
Why did the `private` keyword get removed?

Reply:
Hrmm not sure, I'll change it back.

Reply:
Changed and pushed